packages/blocks/src/api/validation/index.js

```diff
@@ -44,6 +44,7 @@ const REGEXP_STYLE_URL_TYPE = /^url\s*\(['"\s]*(.*?)['"\s]*\)$/;
  * See: https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes
  * Extracted from: https://html.spec.whatwg.org/multipage/indices.html#attributes-3
  *
+ * @example
```

Comment:
This, like the following cases, is not exactly an example of the content being used. But I think can interpret it as an example of the code that was used to extract it. Does it make sense?